packages/account-v2/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "@deriv/account-v2",
+    "version": "2.0.0",
+    "main": "dist/account-v2/js/index.js",
+    "engines": {
+        "node": "18.x"
+    },
+    "scripts": {
+        "build": "rimraf dist && NODE_OPTIONS='-r ts-node/register' webpack --progress --config \"./webpack.config.js\" && npm run translate",
+        "serve": "rimraf dist && concurrently \"cross-env BUILD_MODE='serve' NODE_OPTIONS='-r ts-node/register' webpack --progress --watch --config ./webpack.config.js\" \"tsc -w --noEmit --preserveWatchOutput\"",
+        "start": "rimraf dist && npm run test && npm run serve",
+        "translate": "sh ./scripts/update-translations.sh"
+    },
+    "dependencies": {
+        "@deriv/quill-design": "^1.2.12",
+        "classnames": "^2.3.2",
```

Comment:
we should consider `clsx` which has the same API yet smaller bundle size
https://www.npmjs.com/package/clsx

Reply:
quill design has helper functions & this can be removed completely